operator/src/main/java/oracle/kubernetes/operator/helpers/PodStepContext.java

```diff
@@ -806,6 +824,9 @@ void overrideContainerWeblogicEnvVars(List<V1EnvVar> vars) {
     if (isLocalAdminProtocolChannelSecure()) {
       addEnvVar(vars, "ADMIN_PORT_SECURE", "true");
     }
+    if (isAdminServerProtocolChannelSecure()) {
+      addEnvVar(vars, "ADMIN_SERVER_PORT_SECURE", "true");
```

Comment:
Per discussion, looks like we are creating a new env variable when we already have one.

Also, we should see about being consistent in the naming, for example:
- SERVER_NAME
- SERVER_PORT
- SERVER_PORT_SECURE
- SERVER_SERVICE_NAME
- ADMIN_NAME
- ADMIN_PORT
- ADMIN_PORT_SECURE
- ADMIN_SERVICE_NAME
- ...

and then the scripts can always know the local + admin server content

Reply:
I also realized that changing names may cause upgrade compatibility issue when going operator.current to operator.next so even some doc/readme on the various env vars will help?

Reply:
After revisiting this PR today I think we do need the new ADMIN_SERVER_PORT_SECURE variable (along with the new method) because on the managed server there needs to a property used by the scripts to know if the local managed server port is secure in addition to knowning if the admin server needs to be connected to over a secure port with https.